src/public/javascripts/ui.js

```diff
@@ -1,282 +1,268 @@
-    constsCache.renderInterval = Number(document.getElementById('renderInterval').value);
-    document.getElementById('formFeedback').innerHTML = CONFIG.writeJSON(constsCache);
-    ELECTRON_WINDOW.reload();
-  });
-}
-
-/**
- * Fills constants in settings modal
- */
-function fillConstants() { // eslint-disable-line no-unused-vars
-  CONFIG.updateConstants();
-  document.getElementById('formFeedback').innerHTML = 'Will restart for changes to take place.';
-  document.getElementById('podIP').value = String(CONFIG_CONSTANTS.serverAddr.ip);
-  document.getElementById('podPort').value = CONFIG_CONSTANTS.serverAddr.port;
-  document.getElementById('scanningRate').value = CONFIG_CONSTANTS.dataSendRate;
-  document.getElementById('lvBoneIP').value = CONFIG_CONSTANTS.lvBone.ip;
-  document.getElementById('lvBonePort').value = CONFIG_CONSTANTS.lvBone.port;
-  document.getElementById('hvBoneIP').value = CONFIG_CONSTANTS.hvBone.ip;
-  document.getElementById('hvBonePort').value = CONFIG_CONSTANTS.hvBone.port;
-  document.getElementById('renderInterval').value = CONFIG_CONSTANTS.renderInterval;
-}
-
-// Window Handling
-
-document.getElementById('min-window').addEventListener('click', () => {
-  ELECTRON_WINDOW.minimize();
-});
-
-document.getElementById('max-window').addEventListener('click', () => {
-  if (!ELECTRON_WINDOW.isMaximized()) ELECTRON_WINDOW.maximize();
-  else ELECTRON_WINDOW.unmaximize();
-});
-
-document.getElementById('close-window').addEventListener('click', () => {
-  ELECTRON_WINDOW.close();
-});
+/**
+ * @module UI
+ * @author Eric Udlis, Luke Houge
+ * @description Handles all responsive UI elements of the dashboard
+ */
+
+const { remote: REMOTE } = require('electron');
+
+const ELECTRON_WINDOW = REMOTE.getCurrentWindow();
+const CONFIG = require('./public/javascripts/config');
+const CONFIG_CONSTANTS = require('./public/javascripts/config').constants;
+
+const RATE = CONFIG_CONSTANTS.DATA_SEND_RATE;
+
+/*
+Modals
+Purpose: code for opening a pop up modal box
+*/
+const SETTINGS_MODAL = document.querySelector('.settingsModal');
+const SETTINGS_TRIGGER = document.getElementById('settingsTrigger');
+const CLOSE_BUTTON = document.querySelector('.close-button');
+
+/**
+ * Toggles hide/show of the settings modal
+ */
+function toggleSettingsModal() {
+  SETTINGS_MODAL.classList.toggle('show-modal');
+  fillConstants(); // eslint-disable-line no-use-before-define
+}
+
+if (SETTINGS_TRIGGER) SETTINGS_TRIGGER.addEventListener('click', toggleSettingsModal);
+if (CLOSE_BUTTON) CLOSE_BUTTON.addEventListener('click', toggleSettingsModal);
+
+// window.addEventListener('click', windowOnClick);
+
+/*
+Focus Clone
+Purpose: Fill boxes at the top with live information from tables, for better visibility
+*/
+// Counters for Focus Header
+let focusOne;
+let focusTwo;
+let focusThree;
+let focusFour;
+
+const settingsSubmit = document.getElementById('podSettingsSubmit');
+
+
+// filling for focus clone
+let x = 1; // counter for boxes filed so far

```

Comment:
That's some dirty variable naming

Reply:
@LukeHouge Fix this please, I literally don't know what this does